client/core/bookie.go

```diff
@@ -523,14 +524,18 @@ func handleTradeSuspensionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 	})
 
 	dc.refreshMarkets()
-	c.updateBalances(updatedAssets)
+	if len(updatedAssets) > 0 {
+		c.updateBalances(updatedAssets)
+	} else {
+		c.refreshUser() // would be called by updateBalances
+	}
 
 	return err
 }
 
 // handleTradeResumptionMsg is called when a trade resumption notification is
 // received. This may be an orderbook message at the time of resumption or a
-// general notification at any time prior to the market suspend.
+// general notification at any time prior to market suspend.
 func handleTradeResumptionMsg(c *Core, dc *dexConnection, msg *msgjson.Message) error {
```

Comment:
Bad docs.  Should say `This may be an orderbook message at the time of resumption or notification of a newly-schedule resumption while the market is suspended (prior to the market resume).`